src/core/frames/frame_controller.ts

```diff
@@ -233,11 +234,15 @@ export class FrameController implements AppearanceObserverDelegate, FetchRequest
   // Private
 
   private async visit(url: Locatable) {
-    const request = new FetchRequest(this, FetchMethod.get, expandURL(url), undefined, this.element)
+    const request = new FetchRequest(this, FetchMethod.get, expandURL(url), new URLSearchParams, this.element)
+
+    this.currentFetchRequest?.abortController.abort()
```

Comment:
Could we use `FetchRequest`'s `cancel()` here instead of manually aborting?

https://github.com/hotwired/turbo/blob/3e4becaaf293e5ca6e962f8151c494185241dd17/src/http/fetch_request.ts#L80-L82